### ISSUE TITLE
Several python imports not found

### ISSUE THREAD

**Issue by interface361** [lcfidev/star#19]
Hey,
I just installed python3 with homebrew on my mac, together with the pip module.
I installed the requirements.txt using python3 -m pip install -r requirements.txt, which worked fine.
Now when trying to run the main.py file using the python launcher 3 for macos, it shows me the following missing import errors.
![Bildschirmfoto 2024-08-29 um 16 40 35](https://github.com/user-attachments/assets/42781063-ce57-406e-bf6f-785551a5509c)

Is there anything wrong with the way I tried to run your script ? 
Yours Sincerely,
Interface

**Comment by interface361** [comment]
Additional Info, It only shows the errors for all mt2.(Module) Modules.

**Comment by lcfidev** [comment]
I can suggest a temporary fix which would be remove mt2 from the import and have it as utils.amf

**Comment by interface361** [comment]
I tried to do it with every mt2 import which couldn't be found, still giving errors because It can't find tlsclient-x86.dylib.

I assume I will have to wait until you push a fix for that.

**Comment by lcfidev** [comment]
Yes the tlsclient hasn’t been setup to be dynamic with other platforms yet. Although the code is referenced  in the client src as it’s still very much new

**Comment by interface361** [comment]
Works now, after fixing the imports.

**Comment by interface361** [comment]
Its giving a warning still:

/Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/urllib3/connectionpool.py:1099: InsecureRequestWarning: Unverified HTTPS request is being made to host 'disco.mspapis.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#tls-warnings
  warnings.warn(


And as soon as I try any option within your script, it will instantly say : "press any button to get back to home" without doing the given action.

**Comment by lcfidev** [comment]
That error is fine for now, and only buy_clothes work no other option has been implemented to public yet as it’s a alpha version port of the csharp msptool

**Comment by interface361** [comment]
All good, if you need any more help just let me know. I am also a dev, we are currently working on a KeyAuth replacement. Mainly focusing on security.

**Comment by lcfidev** [comment]
Join discord.gg/hasurei
